sonar-dotnet-shared-library/src/main/java/org/sonar/plugins/dotnet/tests/DotCoverReportParser.java

```diff
@@ -32,11 +32,17 @@
 
 public class DotCoverReportParser implements CoverageParser {
 
+  private static final Pattern TITLE_PATTERN = Pattern.compile(".*?<title>(.*?)</title>.*", Pattern.DOTALL);
+  private static final Pattern COVERED_LINES_PATTERN_1 = Pattern.compile(
+    ".*<script type=\"text/javascript\">\\s*+highlightRanges\\(\\[(.*?)\\]\\);\\s*+</script>.*",
+    Pattern.DOTALL);
+  private static final Pattern COVERED_LINES_PATTERN_2 = Pattern.compile("\\[(\\d++),\\d++,\\d++,\\d++,(\\d++)\\]");
```

Comment:
Possessive quantifiers seems like a overkill (hard to read) for this simple scenario. It doesn't bring value since there's no alternative way of matching this expression.
```suggestion
  private static final Pattern COVERED_LINES_PATTERN_2 = Pattern.compile("\\[(\\d+),\\d+,\\d+,\\d+,(\\d+)\\]");
```

Reply:
I won't change regular expressions in a PR for adding logging - feel free to open a separate PR to do it or a hardening ticket